core/src/nucleus/ribosome/api/init_globals.rs

```diff
@@ -21,22 +12,18 @@ pub fn invoke_init_globals(
     runtime: &mut Runtime,
     _args: &RuntimeArgs,
 ) -> Result<Option<RuntimeValue>, Trap> {
-    let globals = InitGlobalsOutput {
-        app_name: runtime.dna_name.to_string(),
-
+    #[cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
That seems unnecessary. Please don't align the values with spaces or at all. It makes this much harder to maintain